docs/source/user/vectorize.rst

```diff
@@ -260,39 +264,40 @@ Overwriting input values
 ------------------------
 
 In most cases, writing to inputs may also appear to work - however, this
-behaviour cannot be relied on. Consider the following example function::
+behaviour cannot be relied on. Consider the following example function:
 
-   @guvectorize([(float64[:], float64[:])], '()->()')
-   def init_values(invals, outvals):
-       invals[0] = 6.5
-       outvals[0] = 4.2
+.. literalinclude:: ../../../numba/tests/doc_examples/test_examples.py
+   :language: python
+   :caption: from ``test_guvectorize_overwrite`` of ``numba/tests/doc_examples/test_examples.py``
+   :start-after: magictoken.ex_guvectorize_overwrite.begin
+   :end-before: magictoken.ex_guvectorize_overwrite.end
+   :dedent: 12
+   :linenos:
 
 Calling the `init_values` function with an array of `float64` type results in
-visible changes to the input::
+visible changes to the input:
 
-   >>> invals = np.zeros(shape=(3, 3), dtype=np.float64)
-   >>> outvals = init_values(invals)
-   >>> invals
-   array([[6.5, 6.5, 6.5],
-          [6.5, 6.5, 6.5],
-          [6.5, 6.5, 6.5]])
-   >>> outvals
-   array([[4.2, 4.2, 4.2],
-       [4.2, 4.2, 4.2],
-       [4.2, 4.2, 4.2]])
+.. literalinclude:: ../../../numba/tests/doc_examples/test_examples.py
+   :language: python
+   :caption: from ``test_guvectorize_overwrite`` of ``numba/tests/doc_examples/test_examples.py``
+   :start-after: magictoken.ex_guvectorize_overwrite_call_one.begin
+   :end-before: magictoken.ex_guvectorize_overwrite_call_one.end
+   :dedent: 12
+   :linenos:
 
 This works because NumPy can pass the input data directly into the `init_values`
 function as the data `dtype` matches that of the declared argument.  However, it
 may also create and pass in a temporary array, in which case changes to the
 input are lost. For example, this can occur when casting is required. To
-demonstrate, we can  use an array of `float32` with the `init_values` function::
+demonstrate, we can  use an array of `float32` with the `init_values` function:
 
-   >>> invals = np.zeros(shape=(3, 3), dtype=np.float32)
-   >>> outvals = init_values(invals)
-   >>> invals
-   array([[0., 0., 0.],
-          [0., 0., 0.],
-          [0., 0., 0.]], dtype=float32)
+.. literalinclude:: ../../../numba/tests/doc_examples/test_examples.py
+   :language: python
+   :caption: from ``test_guvectorize_overwrite`` of ``numba/tests/doc_examples/test_examples.py``
+   :start-after: magictoken.ex_guvectorize_overwrite_call_two.begin
+   :end-before: magictoken.ex_guvectorize_overwrite_call_two.end
+   :dedent: 12
+   :linenos:
```

Comment:
I can't write a comment on a line unmodified by this PR, but it looks like there is one block below (beginning on line 305 / 310 below) that is unconverted. This block was probably added after you started this PR, which might be why it's unconverted.